document/ToolOrchestrationProfile.mdk

```diff
@@ -0,0 +1,56 @@
+# Tool Orchestration Profile { #tool-orchestration-profile }
+
+## General
+
+It is often desirable to integrate the results produced by a static analysis tool
+into a larger process that orchestrates the operation of several tools. For example,
+static analysis might be part of a nightly build process that combines build tools,
+test tools, and deployment tools.
+
+The Tool Orchestration Profile defines additional optional elements of the SARIF format
+that are intended for use in this scenario.
+
+## `runInfo` object { #tool-orchestration-profile-runInfo }
+
+### General { #tool-orchestration-profile-runInfo-general }
+
+The Tool Orchestration Profile defines four additional properties on the `runInfo` object (&sect;[#runInfo]).
+
+### `correlationId` property { #tool-orchestration-profile-runInfo-correlationId }
+
+A `runInfo` object may contain a property named `correlationId` whose value is a string
+that allows the run to be associated with other artifacts.
+
+~ IsoNote
+In an environment where a static analysis tool is executed as part of a larger
+automated build process, the "build id" might serve as the `correlationId`,
+allowing the tool run to be associated with other artifacts produced by the build.
+~
+
+### `architecture` property { #tool-orchestration-profile-runInfo-architecture }
+
+A `runInfo` object may contain a property named `architecture` that specifies the hardware
```

Comment:
'the hardware architecture at which the analysis targets are targeted'. a bit clumsy, use of target seems redundant.
how about
'the hardware architecture for which the analysis run was targeted' or 'against which'?